docs/_integration-manager/camel-routes/camel-route-aws.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Camel Route Configuration for AWS S3
+nav_order: 2
+parent: File Folder Listener
+---
+# Camel Route Configuration for AWS S3
+
+## Overview
+
+The File Folder Listner can be configured with Camel route and its typically consists of a series of processing steps that are connected in a linear sequence.
+
+## Route Configuration
+
+Note that the File Folder Listener Service must be restarted for any configuration changes to take effect. Make sure you have already completed: File Folder Listener Authorization
```

Comment:
this is no longer relevant - the service does not need to be restarted for camel route changes

Reply:
Done.